server/src/bootstrap/config/auth.ts

```diff
@@ -9,7 +9,7 @@ const authSchema: Schema<AuthConfig> = {
   jwtSecret: {
     doc: 'JWT authentication secret',
     format: String,
-    default: null,
+    default: 'toomanysecrets',
```

Comment:
there shouldn't be a default here... in production, if there is no value, we deliberately want convict to fail.

Reply:
understand, then can we supply github actions with one? Previous commit failed test and lint https://github.com/opengovsg/askgovsg/pull/162/commits/b78f6a16af99442078e3be3d05bf4858be69be44

Reply:
have left a comment on the relevant spot that is causing the problem